scripts/zeus.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#PBS -q q_short
+#PBS -l nodes=3:ppn=4
+#PBS -l walltime=00:02:00
+
+SUPERVISOR_HOSTNAME=`hostname`
+REMOTE_HOSTS=`cat ${PBS_NODEFILE} | uniq | grep -v ${SUPERVISOR_HOSTNAME}`
+
+java -jar supervisor.jar &
```

Comment:
for supervisor add:
-Dakka.remote.netty.tcp.hostname=${SUPERVISOR_HOSTNAME}
-Dworker.nodes=2
-Dworker.cores=4